dashboard/src/components/widget/loader/che-crane.directive.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+/**
+ * Defines a directive for creating Loader
```

Comment:
Defines a directive for creating Loader = Defines a directive for animating iteration process